docs/extensions/info-xml.md

```diff
@@ -313,22 +315,22 @@ Elements acceptable within `<typeInfo>`
 
 | Element | Acceptable instances | Acceptable<br>when |
 | -- | -- | -- |
-| [`<reportUrl>`](#reportUrl) | 0 or 1 | `<extension type="report">` |
-| [`<userNameLabel>`](#userNameLabel) | 0 or 1 | `<extension type="payment">` |
+| [`<billingMode>`](#billingMode) | 0 or 1 | `<extension type="payment">` |
+| [`<isRecur>`](#isRecur) | 0 or 1 | `<extension type="payment">` |
```

Comment:
Most of the params should probably be specified via managed entities rather than info.xml (see eg. https://github.com/mattwire/org.civicrm.smartdebit/blob/master/smartdebit.mgd.php)